apps/web/src/pages/home/index.tsx

```diff
@@ -304,6 +304,24 @@ const InternalHome = observer(() => {
     }
   });
 
+  const [isSafari, setIsSafari] = useState<boolean>(false);
+
+  const BreakText = (variable: string) => {
```

Comment:
I know this is very very unimportant but could you use the JS naming conventions for functions? 🤪

https://www.w3schools.com/js/js_conventions.asp